pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNonPersistentTopics.java

```diff
@@ -53,7 +53,7 @@ private NonPersistentTopics getNonPersistentTopics() {
 
     @Parameters(commandDescription = "Lookup a topic from the current serving broker")
     private class Lookup extends CliCommand {
-        @Parameter(description = "non-persistent://property/cluster/namespace/topic", required = true)
+        @Parameter(description = "non-persistent://tenant/namespaces/topic", required = true)
```

Comment:
In all the repetitions, it should be:
```suggestion
        @Parameter(description = "non-persistent://tenant/namespace/topic", required = true)
```